truss/tests/test_model_inference.py

```diff
@@ -1844,11 +1844,34 @@ async def predict(self, inputs):
         container = tr.docker_run(
             local_port=8090, detach=True, wait_for_server_ready=False
         )
-        time.sleep(1)
+        time.sleep(3)
```

Comment:
`time.sleep(1)` wasn't enough time for me locally, it was very flaky. I upped to 3 here and below, but might not be necessary if CI environment is somehow faster.